.github/workflows/wheels.yml

```diff
@@ -97,6 +97,13 @@ jobs:
   upload_all:
     needs: [build_wheels, make_sdist]
     runs-on: ubuntu-latest
+    # Restrict to the environment set for the trusted publisher
+    environment:
+      name: publish
```

Comment:
I created a "publish" environment with the following protection rules:

![image](https://github.com/scikit-hep/fastjet/assets/5142394/8a9ed088-82a5-4bd9-b819-1e7b76135a88)